apps/k8s-io/README.md

```diff
@@ -57,6 +57,8 @@ Redirections
 - https://go.k8s.io/test-history
 - https://go.k8s.io/triage
 - https://go.k8s.io/logo
+- https://go.k8s.io/contact
+- https://go.k8s.io/contact/${GROUP_NAME}
```

Comment:
Let's also add an example here as a sub-bullet point so that folks know what this redirects to
```suggestion
- https://go.k8s.io/contact/${GROUP_NAME}
  - For example: 
    - https://go.k8s.io/contact/sig-contributor-experience
    - https://go.k8s.io/contact/wg-lts
    - https://go.k8s.io/contact/committee-steering
```